modules/sdk-coin-ton/src/lib/withdrawBuilder.ts

```diff
@@ -0,0 +1,31 @@
+import { TransactionBuilder } from './transactionBuilder';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { Recipient, TransactionType } from '@bitgo/sdk-core';
+import TonWeb from 'tonweb';
+
+export class WithdrawBuilder extends TransactionBuilder {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.Send;
+  }
+
+  send(recipient: Recipient): WithdrawBuilder {
+    this.transaction.recipient = recipient;
+    return this;
+  }
+
+  private createMessage(): string {
+    const opcode = '00001000';
+    const query_id = '0000000000000000';
+    const amount = TonWeb.utils.toNano(this.transaction.recipient.amount);
+    return opcode.concat(query_id).concat(amount.toString('hex'));
```

Comment:
this won't work. Ton is using bag of cell(boc) packing, I think you need to put it in boc structure.